docs/source/private/overview.rst

```diff
@@ -194,31 +194,34 @@ Phase 2: Data Gathering
 What data will we gather?
 =========================
 
-We will need the following data from you:
+Like the public benchmark we need to gather some general data about the alchemical network you have run, more information
+about the script which extracts the data and the contents of the final data package to be shared with the OpenFE team can
+be found :ref:`here <gathering_of_results>`.
 
-* Calculated ddG values.
-* Experimental assay values (micromolar).
-* Broad assay description (e.g. "Kd from ITC").
-* Estimated benchmark difficulty (i.e. "easy", "medium", "hard").
-* Convergence data (MBAR overlap, forward/backwards plots, etc...).
-* Number of completed & failed edges.
-
-  * Including information about any attempts at remediating broken networks due to failed edges.
+As this phase involves private datasets we also require you to disclose the experimental data used to validate the accuracy
+of the simulations in a specific CSV format which is shown here for some example data:
 
-* Simulation time and the number of atoms.
-* Blinded transformation networks (i.e. name of nodes and how they are connected).
-* Transformation information.
+.. csv-table:: Example Data
+   :file: experimental_data.csv
+   :header-rows: 1
 
-  * Mapping scores (Lomap, formal charge, shape overlay, RMSD, number of heavy dummy & core atoms) for each edge.
+.. warning::
+    The names of the ligands must match those used when planning your alchemical network and
+    therefore those in the gathered results. If the names are private and you hid them using the
+    :ref:`data gathering script <gathering_of_results>`, an additional script will be provided which can translate
+    those in the experimental CSV to match the anonymised names.
 
-  * 2D and 3D fingerprint similarity scores (e.g. Tanimoto).
+Notes on generating the CSV data:
 
-  * Number of rotatable bonds & rings.
+* Experimental assay values should be supplied as ``nanomolar affinities``.
```

Comment:
Do we have any concerns about folks converting things badly? My initial view was "let them do it and we assume their values are the ground truth", but I wanted to double check that there's no strong views against this.

Reply:
I am in two minds about this, I expect they have already done this conversion for the dataset assuming they have benchmarked FEP+ and double-checked they have done it correctly. On the other hand, let's ask for the raw results to ensure we support the types of conversions they have done in cinnabar so that this can be done automatically in validation plots in the future.

Reply:
Yeah I see what you mean. Unless @hannahbaumann feels otherwise - let's just stick with nM and let them do their own conversions. We can deal with the cinnabar transformation stuff later (at least a lot of it is already in the PLB so we can take that out and start with it).